engines/agi/detection_tables.h

```diff
@@ -23,15 +23,15 @@
 
 namespace Agi {
 
-#define GAMEOPTIONS_DEFAULT                   GUIO5(GAMEOPTION_ORIGINAL_SAVELOAD,GAMEOPTION_ENABLE_MOUSE,GAMEOPTION_ENABLE_PREDICTIVE_FOR_MOUSE,GAMEOPTION_USE_HERCULES_FONT,GAMEOPTION_COMMAND_PROMPT_WINDOW)
```

Comment:
Should that option be available in commercial AGI games as well, or only for fan made games?

Reply:
I don't have enough commercial games to test, but indeed I'd go with fanmades only, as I'm not sure any commercial one used noise for music.

Reply:
Done.